src/controller/python/chip-device-ctrl.py

```diff
@@ -560,6 +560,74 @@ def do_discover(self, line):
 
         self.devCtrl.PrintDiscoveredDevices()
 
+    def do_ping(self, line):
```

Comment:
Could you update the `README.md` with a new command description under `List of commands` section?

Reply:
Acked